server/eclipse-project/script/init-db

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+readonly __dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+readonly cmd="psql -d corla -a -f "${__dir}/insert-test-creds.sql""
```

Comment:
(and they will definitely have to change, once we make authentication more robust)